utils/suggestions.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {AsciiToUtf8Characters} from './constants';
+
+export const asciiToUtf8Variants = (term: string) => {
+    let variants = [term];
+    term.split('').forEach((t: string, i: number) => {
+        if ((AsciiToUtf8Characters as {[key: string]: string[]})[t] && i < 5) {
```

Comment:
Weird that TS can't figure out the type of this on its own. You could get rid of one of the `as`s though if you give this value a name